administration/src/bp-modules/users/RoleSelector.tsx

```diff
@@ -12,14 +13,21 @@ const RoleSelector = ({
   role: Role | null
   onChange: (role: Role | null) => void
   hideProjectAdmin: boolean
-}): ReactElement => (
-  <HTMLSelect fill onChange={e => onChange((e.target.value as Role | null) ?? null)} value={role ?? ''} required>
-    <option value='' disabled>
-      Auswählen...
-    </option>
-    {hideProjectAdmin ? null : <option value={Role.ProjectAdmin}>{roleToText(Role.ProjectAdmin)}</option>}
-    <option value={Role.RegionAdmin}>{roleToText(Role.RegionAdmin)}</option>
-    <option value={Role.RegionManager}>{roleToText(Role.RegionManager)}</option>
-  </HTMLSelect>
-)
+}): ReactElement => {
+  const config = useContext(ProjectConfigContext)
+
+  return (
+    <HTMLSelect fill onChange={e => onChange((e.target.value as Role | null) ?? null)} value={role ?? ''} required>
+      <option value='' disabled>
+        Auswählen...
+      </option>
+      {hideProjectAdmin ? null : <option value={Role.ProjectAdmin}>{roleToText(Role.ProjectAdmin)}</option>}
+      {hideProjectAdmin || !config.applicationFeature?.applicationUsableWithApiToken ? null : (
+        <option value={Role.ExternalVerifiedApiUser}>{roleToText(Role.ExternalVerifiedApiUser)}</option>
```

Comment:
```suggestion
     {config.applicationFeature?.applicationUsableWithApiToken && currentUserRole === Role.ProjectAdmin ? (
        <option value={Role.ExternalVerifiedApiUser}>{roleToText(Role.ExternalVerifiedApiUser)}</option>
      ) : null}
```
I think your check is really hard to read and even we need a line code more i think we should add the context for the currentUserRole and check that
```js
const { role: currentUserRole } = useContext(WhoAmIContext).me!
```